src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$width-boxs: 100px; // Ширина одного дня
+$width-gap: 1px; // Проміжок між днями
+$calendar-days: 31; // Максимальна кількість днів у місяці
+$days-of-the-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+); // Відображення днів тижня
```

Comment:
There is an unnecessary comma at the end of the `$days-of-the-week` map declaration. While this is not a syntax error in SCSS, it is cleaner to remove it to adhere to the code style guidelines.